web/docs/packages.md

```diff
@@ -153,3 +168,67 @@ contexts:
 :::tip Use packages
 Start using packages by [installing one](installation/install-a-package.md).
 :::
+
+## User configuration
+
+In order to install a package from the library, the user usually has to provide
```

Comment:
We try to use active voice and present tense in documentation where possible. Most of the newly added documentation in this PR is written in passive voice. Can you make a sweeping pass over this to change it?